.github/workflows/typecheck.yml

```
@@ -20,3 +20,12 @@ jobs:
         run: npm run typecheck
         env:
           CI: true
+
+      - name: Check for new JavaScript files
+        run: |
+          git fetch origin main --no-tags --depth=1
+          count_new_js=$(git diff --name-only --diff-filter=A origin/main HEAD -- 'src/libs/*.js' 'src/hooks/*.js' 'src/styles/*.js' 'src/languages/*.js' | wc -l)
```

```suggestion
          count_new_js=$(git diff --name-only --diff-filter=A origin/main HEAD -- 'src/libs/**/*.js' 'src/hooks/**/*.js' 'src/styles/**/*.js' 'src/languages/**/*.js' | wc -l)
```

I believe it should be this way to consider files in nested folders?